packages/developer-portal/src/components/iaas/index.tsx

```diff
@@ -74,47 +72,35 @@ export const IaaS: FC = () => {
       </SecondaryNavContainer>
       <PageContainer>
         <ErrorBoundary>
-          <PusherProvider
-            cluster="eu"
-            clientKey={window.reapit.config.PUSHER_KEY}
-            authEndpoint={`${URLS.DEPLOYMENT_SERVICE_HOST}pusher/auth`}
-            auth={{
-              headers: {
-                ...COGNITO_HEADERS,
-                Authorization: connectSession.idToken,
-              },
-            }}
-          >
-            <Title>Pipelines</Title>
-            {loading || !currentDeveloper ? (
-              <Loader />
-            ) : (
-              <>
-                <Table data-has-expandable-action data-num-columns-excl-action-col="3">
-                  <TableHeadersRow>
-                    <TableHeader>Name</TableHeader>
-                    <TableHeader>Status</TableHeader>
-                    <TableHeader>Repository</TableHeader>
-                    <TableHeader></TableHeader>
-                  </TableHeadersRow>
-                  {pagination?.items?.map((pipeline) => (
-                    <PipelineRow connectSession={connectSession} pipeline={pipeline} key={pipeline.id} />
-                  ))}
-                </Table>
-                <div className={elMt6}>
-                  {pagination && (
-                    <Pagination
-                      currentPage={pagination.meta.currentPage}
-                      numberPages={pagination.meta.totalPages}
-                      callback={(page) => {
-                        setPage(page)
-                      }}
-                    />
-                  )}
-                </div>
-              </>
-            )}
-          </PusherProvider>
+          <Title>Pipelines</Title>
```

Comment:
The table row component has a useEvent listener. If we remove this provider it will stop working, or even maybe blow up right?

Reply:
yea just noticed and rebased

Reply:
I can't see how you resolved this, looks the same in the diff

Reply:
my bad, I thougt there was a global provider for pusher

Reply:
ma bad, I thought there was a global wrapper but it only applies to apps